virttest/qemu_vm.py

```diff
@@ -992,7 +992,7 @@ def set_value(opt_string, key, fallback=None):
                 else:
                     self.spice_options['spice_port'] = self.spice_port
                     spice_opts.append("port=%s" % self.spice_port)
-            else:
+            elif optget("spice_port") != "no":
```

Comment:
For clarity's sake I would ask to add a comment:
"""
spice_port = no, means that 'spice_port' parameter is absent in qemu cmdline.
"""
It takes a time to understand what "no" means.
Having a such comment greatly helps to understand what is "no".